codelists/models.py

```diff
@@ -313,6 +313,13 @@ def get_diff_url(self, other_clv):
         kwargs["other_tag_or_hash"] = other_clv.tag_or_hash
         return reverse(f"codelists:{self.codelist_type}_version_diff", kwargs=kwargs)
 
+    def exists(self):
```

Comment:
Would prefer it if this was called `has_any_versions`.  It's a bit long-winded, but it communicates the intent more clearly.

Reply:
Also: I can't see where this is used -- am I missing something?

Reply:
This method is on a `CodelistVersion` though, so `has_any_versions` doesn't seem like the right name.  It's used here: https://github.com/opensafely-core/opencodelists/blob/main/builder/decorators.py#L20

Reply:
Oh sorry, my mistake!